neoexchange/neox/tests/test_plot_lc.py

```diff
@@ -118,3 +119,16 @@ def test_can_view_lightcurve(self):   # test opening up a ALCDEF file associated
         period_box = self.browser.find_element_by_xpath("/html/body[@class='page']/div[@id='page-wrapper']/div[@id='page']/div[@id='main']/div[@name='lc_plot']/div[@class='bk']/div[@class='bk'][2]/div[@class='bk'][2]/div[@class='bk'][2]/div[@class='bk']/div[@class='bk'][1]/div[@class='bk'][1]/div[@class='bk']/div[@class='bk bk-input-group']/div[@class='bk bk-spin-wrapper']/input[@class='bk bk-input']")
         period_text = self.browser.find_element_by_xpath("/html/body[@class='page']/div[@id='page-wrapper']/div[@id='page']/div[@id='main']/div[@name='lc_plot']/div[@class='bk']/div[@class='bk'][2]/div[@class='bk'][2]/div[@class='bk'][2]/div[@class='bk']/div[@class='bk'][1]/div[@class='bk'][1]/div[@class='bk']/div[@class='bk bk-input-group']/label[@class='bk']").text
         self.assertIn('Period (Default: 5.27h)', period_text)
+        # get current window
+        pw = self.browser.current_window_handle
+        data_link = self.browser.find_element_by_xpath("/html/body/div[1]/div/div[3]/div/div/div[2]/div[2]/div[1]/div[2]/div[4]/div[3]/div/div[1]/div[7]/a")
+        data_link.click()
+        time.sleep(1)
```

Comment:
I think `time.sleep(1)` IS how you do an explicit wait for selenium...

Getting an implicit wait to function for a second window/tab was throwing me off because you have to wait twice!
you can't just wait for the tab to open you have to THEN wait for it to load once you switch windows.

Fixed now.